scripts/inject.js

```diff
@@ -42,7 +42,6 @@ const builds = {
         output: [
             'build/integration/contentScope.js',
             'integration-test/extension/contentScope.js',
-            'integration-test/pages/build/contentScope.js',
```

Comment:
Ah did we do this process twice? The rationale for the separation for pages/ and test-pages/ was so that test-pages explicitly are shared into privacy-test pages where as the former are little helpers kinda for these legacy tests. (where possible the test-pages pattern should be used in my mind).

I don't mind with blending the two directories, so long as you're aware this directory is also going to be served as a web page: https://privacy-test-pages.site/content-scope-scripts/ (we could make the index.html a directory file for example and it be helpful anyway)